shell/app/common/components/key-value-list.tsx

```diff
@@ -79,7 +82,14 @@ export const KeyValueList = ({
                 return (
                   <div className="k-v-row" key={normalK}>
                     <span className="key">{normalK}</span>
-                    <span className="value"> {textRender ? textRender(normalK, normalV) : String(normalV)} </span>
+                    {normalK === 'desc' && withMarkdown ? (
```

Comment:
since this is a top level common component, if we write `desc` here means it involves the business logic, which I think is not good practice.  I suggest replace `withMarkdown` to `markdownTextFields` which is string array. Then pass `markdownTextFields = ['desc']` to implement the same function